package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ekstra-bladet/designsystem",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
for fremtiden må du gerne lade være med at sætte version da det er nemmere for mig og @tegner at gøre det med yarn publish

Reply:
ok

Reply:
@tcarlsen Så generelt sætter vi ikke selv version i .json mere i designsystemet?

Reply:
Ikke i PR's nej